app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/triage/singledisposals/summary.scala.html

```diff
@@ -44,11 +45,19 @@
 @isAgent = @{ request.userType.contains(UserType.Agent) }
 
 @userKey = @{
-    if (representativeType.exists(_.isLeft)) ".personalRep"
-    else if (isAgent) ".agent"
-    else if (isATrust) ".trust"
-    else if (representativeType.exists(_.isRight)) ".capacitor"
-    else ""
+    if(representativeType.exists(_.isLeft)) ".personalRep"
+    else if(isAgent) ".agent"
+    else if(isATrust) ".trust"
+        else if(representativeType.exists(_.isRight)) ".capacitor"
+            else ""
+}
+
+@isSingleDateDisplay = @{
```

Comment:
`isIndirecDisposal` would be a better name here

Reply:
also
```scala
@x = @{
  val y = blah
  y
} 
```
is equivalent to just
```scala
@x = @{
  blah
}
```

Reply:
Done.